pkg/manifests/manifests.go

```diff
@@ -2610,6 +2674,45 @@ func (f *Factory) HashTrustedCA(caBundleCM *v1.ConfigMap, prefix string) *v1.Con
 	}
 }
 
+// HashSecret synthesizes a secret by setting the given data
```

Comment:
Hmm this sentence sounds odd to me ):

Reply:
indeed :-) I will change this as a follow up.